cpp/include/ray/api/object_ref.h

```diff
@@ -83,7 +91,7 @@ const ObjectID &ObjectRef<T>::ID() const {
 
 template <typename T>
 inline std::shared_ptr<T> ObjectRef<T>::Get() const {
-  return Ray::Get(*this);
+  return ray::api::Get(*this);
```

Comment:
Is this change relevant with circular references?

Reply:
Yes, the  Ray::Get in api.h and ObjectRef::Get in object_ref.h both need to invoke this ray::api::Get, so move the Get into the object_ref.h to avoid circular references. 

The ray::api::Get is an internal common function, maybe change the name to GetFromRuntime to distinguish Ray::Get is better.